va/va.h

```diff
@@ -4187,6 +4187,25 @@ Synchronization
  * This function blocks until all pending operations on the render target
  * have been completed.  Upon return it is safe to use the render target for a
  * different picture.
+ *
+ * Possible errors:
+ * - \ref VA_STATUS_ERROR_INVALID_CONTEXT: This error occurs when the function is provided with an invalid context.
+ *   It is typically due to the context not being properly initialized or already being freed.
+ *   To address this, ensure that the context is correctly initialized and has not been freed.
+ * - \ref VA_STATUS_ERROR_INVALID_SURFACE: This error is triggered when the supplied render target is invalid,
+ *   possibly due to being freed or corrupted. To resolve this, ensure the usage of the correct render target
+ *   and verify its validity and integrity.
+ * - \ref VA_STATUS_ERROR_OPERATION_FAILED: When the attempt to retrieve the status report fails or related work
+ *   on the current surface fails, this error is generated. In such cases, the client can proceed with subsequent
+ *   frames by ignore the error, but it may involve incorrect result like corruption.
+ * - \ref VA_STATUS_ERROR_HW_BUSY: This error indicates that synchronization is still in progress or GPU hang happens.
+ *   The client could call the function again within a period of time to wait hw execution completion.
+ *   If hw execution could not complete successfully, gpu hang and reset may occur, in this case,
+ *   application could try to recover media context by initialing new one.
```

Comment:
must recreate context? could application just ignore it?

Reply:
I think the recreate context is must, VA_STATUS_ERROR_HW_BUSY indicate that HW reset happened, this need to recover media context. Media reset will not return this error.